amethyst_renderer/src/pass/shaded_util.rs

```diff
@@ -35,16 +37,17 @@ pub(crate) fn set_light_args(
     effect: &mut Effect,
     encoder: &mut Encoder,
     light: &ReadStorage<Light>,
+    global: &ReadStorage<GlobalTransform>,
     ambient: &AmbientColor,
     camera: Option<(&Camera, &GlobalTransform)>,
 ) {
-    let point_lights: Vec<_> = light
+    let point_lights: Vec<_> = (light, global)
         .join()
-        .filter_map(|light| {
+        .filter_map(|(light, transform)| {
             if let Light::Point(ref light) = *light {
                 Some(
                     PointLightPod {
-                        position: light.center.into(),
+                        position: (transform.0 * Vector4::new(0.0, 0.0, 0.0, 1.0)).truncate().into(),
```

Comment:
I'm pretty sure we should remove this position field as well

Reply:
I don't think so because this is where the data is actually fed into the shaders. The shaders rely on having an array of data about each light and there's not really a point in having them take an actual matrix for a point light in which the rotation/scale doesn't actually affect anything about it. If we fed the matrix into the shader then we'd have to do this matrix -> position conversion for every fragment for every light.

Reply:
I agree, we should feed it like this. But can we instead use transform_point